### ISSUE TITLE
Autogenerate certificate

### ISSUE THREAD

**Issue by Floppy** [theodi/metrics-api#71]

**Comment by pezholio** [comment]
I think the issue with this is generating users. We can do this, but we'd need to generate a user too, which may be problematic from a UX POV. This is worth doing (for Bothan as well as Certificates), but might be too big a job for a Friday afternoon

**Comment by quadrophobiac** [comment]
Hey @Floppy & @pezholio  I've added the link to this ticket to the roadmap card of the same gist.  The current summary of Bothan on the roadmap states
>includes licensing metadata for simple generation of an Open Data Certificate
so I'd like to know whether all the metadata requisite for an Open Data Certificate generation is already within bothan and just needs to be exposed. In that way we can determine if this ticket's workload factors more on the determining the right UX for capturing a user for the cert, or if there is additional work to be done in terms of the metadata capture? (totally asking this as a cheatsheet for what's required for ODC too btw)

**Comment by pezholio** [comment]
Yep, it's all within the Bothan homepage in the DCAT RDFa, so Certificates should be able to glean enough info to generate a basic certificate. The challenge is getting a user set up with an ODCs username and password.

**Comment by quadrophobiac** [comment]
excellent gents, TY for the specification!
